docs/usage/spider-subnet.md

```diff
@@ -19,6 +19,8 @@ To allocate fixed IP addresses for applications and decouple the roles of applic
 
 SpiderSubnet also supports several controllers, including ReplicaSet, Deployment, StatefulSet, DaemonSet, Job, CronJob, and k8s extended operator. If you need to use a third-party controller, you can refer to the doc [Spiderpool supports operator](./operator.md).
 
+This feature does not support the bare Pod.
```

Comment:
The text looks good to me. In future, we can add a link or an example by referring to [k8s.io docs](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/#non-template-pod-acquisitions)